src/Components/List/Card.scss

```
@@ -0,0 +1,70 @@
+@import "../../Styles/common.scss";
+@import "../../Styles/reset.scss";
+
+.Card {
+    width: 255px;
+    height: 383px;
+    margin-right: 20px;
+    display: list-item;
+    position: relative;
+    .thumnailWrap {
+        width: 255px;
+        height: 255px;
+        display: block;
+        position: relative;
+        border-radius: 6px;
+        overflow: hidden;
+        .imgWrap {
+            width: 100%;
+            display: inline-block;
+            position: absolute;
+            top: 0px;
+            left: 0px;
+            right: 0px;
+            bottom: 0px;
+            padding-top: 100%;
+            content: "";
+            background: rgba(0, 0, 0, 0.02);
+            z-index: 0;
+            &::after {
+                content: "";
+            }
+        }
+        img {
+            width: 100%;
+            height: auto;
+            z-index: -1;
+        }
+    }
+    p {
+        font-size: 16px;
+        &.productName {
+            margin: 10px 0 5px 0;
```

```suggestion
            margin-top: 10px;
            margin-bottom: 5px;
```
이 경우 직접 명시하는 방법이 더 좋아보입니다.